rangeseekbar/src/main/java/org/florescu/android/rangeseekbar/RangeSeekBar.java

```diff
@@ -606,8 +606,7 @@ protected synchronized void onDraw(@NonNull Canvas canvas) {
         mRect.right = getWidth() - padding;
         canvas.drawRect(mRect, paint);
 
-        boolean selectedValuesAreDefault = (getSelectedMinValue().equals(getAbsoluteMinValue()) &&
-                getSelectedMaxValue().equals(getAbsoluteMaxValue()));
+        boolean selectedValuesAreDefault = (normalizedMinValue < 0.00001 && normalizedMaxValue > 0.99999);
```

Comment:
Yes, it could be solved as you suggest. It would be more robust, and work for my use-case as well. I'll fix it in a moment then.
A fixed delta of 0.00001 is IMHO safe as well - because it's always much less than the size of one pixel (that's why I switched to normalized values).